src/Compilers/VisualBasic/Portable/Symbols/Source/SynthesizedFieldSymbol.vb

```diff
@@ -59,6 +59,14 @@ Namespace Microsoft.CodeAnalysis.VisualBasic.Symbols
                 Return
             End If
 
+            Dim compilation = Me.DeclaringCompilation
+            If Type.ContainsTupleNames() AndAlso
+                compilation.HasTupleNamesAttributes AndAlso
```

Comment:
IIRC, I think that diagnostic is reported elsewhere. This check prevents errors, should we ever get this far (which I believe is possible in the APIs).

---
In reply to: [79716733](https://github.com/dotnet/roslyn/pull/13923#discussion_r79716733) [](ancestors = 79716733,79676084)